src/consensus/pbft/libbyz/Req_queue.h

```diff
@@ -103,10 +106,18 @@ inline int Req_queue::num_bytes() const
 
 inline Request* Req_queue::first() const
```

Comment:
maybe instead of declaring `rnodes` and `count` as mutable we could just remove `const` from the `first()` method

Reply:
I think the correct fix is to make get_head() const. I will fix this in a later PR as this will involve updating snmalloc